### PR TITLE
Software-manager adjustments

### PR DESCRIPTION
**Pull request by willianrampazzo**
Adjusts the yum backend log level from the yum module and makes log messages available to others.

### CODE REVIEW

File: avocado/utils/software_manager/backends/yum.py [hunk 1]
```diff
@@ -45,7 +45,7 @@ def __init__(self, cmd='yum'):
             self.yum_base = yum.YumBase()
         else:
             self.yum_base = None
-            log.error("%s module for Python is required. "
+            log.debug("%s module for Python is required. "
```
Comment (clebergnu) [inline avocado/utils/software_manager/backends/yum.py:48]: Yes, it says it's required, but still, all the methods like `install`, `remove`, etc, rely on calling the `yum` commands.  Bonus points for changing the wordings, and properly describing what's gained by using the `yum` Python module.

Comment (willianrampazzo) [reply avocado/utils/software_manager/backends/yum.py:48]: That's true. I'll update the message.

**Review by clebergnu — APPROVED** [review]
LGTM (with or without the changes to the message).

File: avocado/utils/software_manager/backends/yum.py [hunk 2]
```diff
@@ -45,9 +46,9 @@ def __init__(self, cmd='yum'):
             self.yum_base = yum.YumBase()
         else:
             self.yum_base = None
-            log.error("%s module for Python is required. "
-                      "Using the basic support from rpm and %s commands", cmd,
-                      cmd)
+            log.debug("%s module for Python is required to use the 'provides'"
```
Comment (clebergnu) [inline avocado/utils/software_manager/backends/yum.py:49]: Yay! Thanks!

**Review by richtja — APPROVED** [review]

**Review by beraldoleal — APPROVED** [review]

**Pull request merged**